lib/components/Form/Input/index.jsx

```diff
@@ -20,6 +21,7 @@ const Input = ({
       defaultValue={value}
       autoComplete={autoComplete}
       required={required}
+      readOnly={readOnly ? 'readOnly' : ''}
```

Comment:
better to have `undefined` rather than `''` to prevent and empty attribute being put in HTML.

Reply:
react would not do that, rendered when readOnly is not supplied:

<input type="text" id="includes" class="form-control" autocomplete="on" value="">